integration-tests/bats/branch.bats

```diff
@@ -143,4 +143,84 @@ teardown() {
     run dolt branch -D main
     [ "$status" -ne 0 ]
     [[ "$output" =~ "Cannot delete checked out branch 'main'" ]] || false
-}
\ No newline at end of file
+}
+
+@test "branch: supplying multiple directives results in an error" {
+    run dolt branch -m -c main main2
+    [ "$status" -ne 0 ]
+    [[ "$output" =~ "Must specify exactly one of --move/-m, --copy/-c, --delete/-d, -D, --show-current, or --list." ]] || false
+}
+
+@test "branch: -a can only be supplied when listing branches" {
+    run dolt branch -a
+    [ "$status" -eq 0 ]
+
+    run dolt branch -a --list main
+    [ "$status" -eq 0 ]
+
+    dolt branch test
+    run dolt branch -a -d test
+    [ "$status" -ne 0 ]
+    [[ "$output" =~ "--all/-a can only be supplied when listing branches, not when deleting branches" ]] || false
+
+    run dolt branch -a -c copy
+    [ "$status" -ne 0 ]
+    [[ "$output" =~ "--all/-a can only be supplied when listing branches, not when copying branches" ]] || false
+
+    run dolt branch -a -m main new
+    [ "$status" -ne 0 ]
+    [[ "$output" =~ "--all/-a can only be supplied when listing branches, not when moving branches" ]] || false
+
+    run dolt branch -a new
+    [ "$status" -ne 0 ]
+    [[ "$output" =~ "--all/-a can only be supplied when listing branches, not when creating branches" ]] || false
+}
+
+@test "branch: -v can only be supplied when listing branches" {
+    run dolt branch -v
```

Comment:
If you want to run a command and you expect it to return 0, it's better to just run it bare. There is more debugging information printed when it fails.

`dolt branch -v`

(and then can skip checking the status).

Reply:
Oh right, I knew that.

Fixed.